ci/authn-azure/run-authn-azure.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash -ex
```

Comment:
Write to all your bash scripts the following header:
#!/bin/bash
set -euo pipefail

for explanation: https://vaneyckt.io/posts/safer_bash_scripts_with_set_euxo_pipefail/